hail/src/test/scala/is/hail/HailSuite.scala

```diff
@@ -18,21 +21,26 @@ object SparkSuite {
           blockSize = 0)
           .set("spark.unsafe.exceptionOnMemoryLeak", "true")),
       logFile = "/tmp/hail.log")
+
+
+  lazy val hc: HailContext = {
+    val schedulerHost = System.getenv("HAIL_TEST_SCHEDULER_HOST")
+    val hc = if (schedulerHost == null)
+      withSparkBackend()
+    else withDistributedBackend(schedulerHost)
```

Comment:
Sorry, nit: I think our style is to have everything on the same line, or both then and else are on their own lines/blocks.